numpy/core/multiarray.pyi

```diff
@@ -74,6 +74,24 @@ _ArrayLike = _NestedSequence[_SupportsArray[dtype[_SCT]]]
 
 __all__: List[str]
 
+ALLOW_THREADS: Final[int]  # 0 or 1 (system-specific)
+BUFSIZE: L[8192]
+CLIP: L[0]
+WRAP: L[1]
+RAISE: L[2]
+MAXDIMS: L[32]
+MAY_SHARE_BOUNDS: L[0]
+MAY_SHARE_EXACT: L[-1]
```

Comment:
Literal values were taken from their `NPY_<x>`-based counterparts, for example:
https://github.com/numpy/numpy/blob/341736a4e5f0023c8352a553b84d59ba9dd3356d/numpy/core/src/common/mem_overlap.h#L11-L12